content/comm/conferences/index.en.md

```diff
@@ -0,0 +1,89 @@
+---
+title: "R-Ladies Conference Participation"
+menuTitle: "Conference Participation"
+weight: 110
+---
+
+R-Ladies participates in conference sessions such as contributed talks,
+panel discussions, networking events, poster presentations, and booths to 
+increase awareness of our organization.
+
+## Organizing a conference event
+
+If you have an R-Ladies member interested in leading a conference
+session, please notify us at `info@rladies.org` for our records and so that
```

Comment:
If there's support (e.g. paying registration fees) for R-Ladies members who are interested in leading a conference, it might be useful to add it here. If not, ignore this!

Reply:
I was thinking this, too!

Reply:
Still getting the hang of review mode! I added a [commit ](https://github.com/rladies/rladiesguide/pull/72/commits/754026ce3c51887d1c3f1bb53eea5184aa434526) w/ 

> We can financially support the speaker's conference attendance by... <please 
describe>.